pkg/storaged/details.js

```diff
@@ -764,10 +764,26 @@ define([
                 if (!vgroup)
                     return;
 
+                function find_lvol(name) {
+                    var lvols = client.vgroups_lvols[path];
+                    for (var i = 0; i < lvols.length; i++)
```

Comment:
Even though they're not strictly necessary, it's clearer when we use braces around multiline blocks.